string name
string connection_id
string address
uint16 port
string return_address
uint16 return_port
uint32 maximum_bytes_per_second
---